tinygrad/tensor.py

```diff
@@ -379,8 +379,14 @@ def _reduce(self, fxn:Type[Function], axis:Optional[Union[int, Tuple[int, ...]]]
     return ret if keepdim else ret.reshape(shape=shape)
 
   def sum(self, axis=None, keepdim=False): return self._reduce(mlops.Sum, axis, keepdim)
-  def max(self, axis=None, keepdim=False): return self._reduce(mlops.Max, axis, keepdim)
   def min(self, axis=None, keepdim=False): return -((-self).max(axis=axis, keepdim=keepdim))
+  def max(self, axis=None, keepdim=False, ignore_nan=True):
```

Comment:
What does torch do? Do they have `ignore_nan`?

Reply:
I found this:
(https://pytorch.org/docs/stable/autograd.html#torch.autograd.detect_anomaly)

Seems useful for granularity.